packages/brick-container/src/styles/theme/default.css

```
@@ -0,0 +1,115 @@
+:root,
+[data-override-theme="light"] {
+  /* Basic CSS variables Refs: https://tapd.easyops.local/pages/viewpage.action?pageId=36045937 */
+  --color-brand: #0071eb;
+  --color-brand-hover: var(--light-palette-blue-4);
+  --color-brand-active: var(--light-palette-blue-8);
+  --color-fill-global-bg-1: #f7f8fa;
+  --color-fill-global-bg-2: var(--light-palette-gray-1);
+  --color-fill-bg-container-L1: var(--light-palette-gray-1);
+  --color-fill-bg-container-L2: var(--light-palette-gray-1);
+  --color-fill-bg-container-L3: var(--light-palette-gray-1);
+  --color-fill-bg-container-L4: var(--light-palette-gray-1);
+
+  --color-fill-bg-base-1: var(--light-palette-gray-3);
+  --color-fill-bg-base-2: #d9ffffff;
```

最后两位 `ff` 是多余的，不要写，否则还可能有[兼容性问题](https://caniuse.com/css-rrggbbaa)。

```suggestion
  --color-fill-bg-base-2: #d9ffff;
```